common-api/src/main/java/com/koval/resolver/common/api/configuration/ConfigurationManager.java

```diff
@@ -11,14 +15,17 @@
 public final class ConfigurationManager {
 
   private static final String CONFIGURATION_FILE = "configuration.yml";
+  private static final Charset UTF_8 = StandardCharsets.UTF_8;
 
   private ConfigurationManager() {
   }
 
   public static Configuration getConfiguration() {
     Yaml yaml = new Yaml(new Constructor(Configuration.class));
-    try (InputStream inputStream = Configuration.class.getClassLoader().getResourceAsStream(CONFIGURATION_FILE)) {
-      return yaml.load(inputStream);
+    try (InputStream inputStream = Configuration.class.getClassLoader().getResourceAsStream(CONFIGURATION_FILE);
```

Comment:
There's no reason to use BufferedReader here, so please revert all changes for ConfigurationManager class

Reply:
Ok